src/App/Fossa/VPS/Scan.hs

```diff
@@ -43,95 +39,22 @@ scanMain basedir apiOpts metadata logSeverity overrideProject fileFilters skipIp
 vpsScan ::
   ( Has Diagnostics sig m
   , Has (Lift IO) sig m
-  ) => BaseDir -> Severity -> OverrideProject -> Flag SkipIPRScan -> FilterExpressions -> ApiOpts -> ProjectMetadata -> BinaryPaths -> m ()
-vpsScan (BaseDir basedir) logSeverity overrideProject skipIprFlag fileFilters apiOpts metadata binaryPaths = withLogQueue logSeverity $ \queue -> runLogger queue $ do
-  -- Build the revision
-  ProjectRevision {..} <- mergeOverride overrideProject <$> inferProject basedir
-
-  -- Get Sherlock info
-  logDebug "[Sherlock] Retrieving Sherlock information from FOSSA"
-  SherlockInfo{..} <- getSherlockInfo apiOpts
-
-  -- Build locator info
-  let locator = createLocator projectName sherlockOrgId
-  let revisionLocator = createRevisionLocator projectName sherlockOrgId projectRevision
-
-  -- FIXME: use better type here
-  let vpsOpts = VPSOpts projectName (Just projectRevision) (fromFlag SkipIPRScan skipIprFlag) fileFilters
-
-  -- Update vpsOpts with overriding scan filter blob. 
-  -- Previous uses of `vpsOpts` do not deconstruct the object due to this not yet being overridden.
-  (vpsOpts'@VPSOpts{..}, areFiltersOverridden) <- overrideScanFilters apiOpts vpsOpts locator
-  logDebug $ pretty $ "[All] Creating project with locator '" <> unRevisionLocator revisionLocator <> "'"
-
-  -- Create scan in Core
-  logDebug "[All] Creating project in FOSSA"
-  _ <- context "creating project in FOSSA" $ createCoreProject vpsProjectName projectRevision metadata apiOpts
-
-  -- Create scan in SY
-  logDebug "[All] Creating scan in Scotland Yard"
-  let syOpts = ScotlandYardOpts locator projectRevision sherlockOrgId vpsOpts'
-  response <- context "creating scan ID" $ createScotlandYardScan apiOpts syOpts
-  let scanId = createScanResponseId response
+  ) => BaseDir -> Severity -> OverrideProject -> Flag SkipIPRScan -> Flag LicenseOnlyScan -> FilterExpressions -> ApiOpts -> ProjectMetadata -> BinaryPaths -> m ()
+vpsScan (BaseDir basedir) logSeverity overrideProject skipIprFlag licenseOnlyScan fileFilters apiOpts metadata binaryPaths = withLogQueue logSeverity $ \queue -> runLogger queue $ do
```

Comment:
We can simplify the logging call here, since we're no longer using `concurrently`.

```suggestion
vpsScan (BaseDir basedir) logSeverity overrideProject skipIprFlag licenseOnlyScan fileFilters apiOpts metadata binaryPaths = withLogger logSeverity $ do
```

Reply:
Made this change and the one in your other outstanding comment and both error and non-error cases work great! Thanks!